xcodeproj/xcodeproj.doc.bzl

```diff
@@ -0,0 +1,17 @@
+"""Public rules, macros, and libraries."""
+
+load(
+    "//xcodeproj/internal:providers.bzl",
+    _XcodeProjAutomaticTargetProcessingInfo = "XcodeProjAutomaticTargetProcessingInfo",
+)
+load("//xcodeproj/internal:xcode_schemes.bzl", _xcode_schemes = "xcode_schemes")
+load("//xcodeproj/internal:xcodeproj_rule.bzl", _xcodeproj = "xcodeproj")
+
+# Re-exporting providers
```

Comment:
We can leave all of the comments off in this file, making it one big blob like in rules_apple.